modules/phase_field/include/kernels/ACBulk.h

```diff
@@ -30,8 +33,19 @@ class ACBulk : public KernelValue
   virtual Real precomputeQpResidual();
   virtual Real precomputeQpJacobian();
   virtual Real computeDFDOP(PFFunctionType type) = 0;
+  virtual Real computeQpOffDiagJacobian(unsigned int jvar);
+
+  /// Name of mobility material property
+  const MaterialPropertyName _mob_name;
 
+  /// Mobility
   const MaterialProperty<Real> & _L;
+
+  /// Mobility derivatives w.r.t. order parameter
+  const MaterialProperty<Real> & _dLdop;
+
+  /// Mobility derivative w.r.t coupled variables
+  std::vector<const MaterialProperty<Real> *> _dLdarg;
```

Comment:
Since this appears in both ```ACBulk``` and ```ACInterface``` should we maybe factor this out into a common base class?